api/event/discovery.go

```diff
@@ -32,13 +32,22 @@ var (
 		ID:   ShardEventKindVersion.String(),
 		Type: bus.ChTypeUnidirectional,
 	}
+	SeriesEventKindVersion = common.KindVersion{
+		Version: "v1",
+		Kind:    "event-series",
+	}
+	TopicSeriesEvent = bus.Topic{
```

Comment:
Since we have `bus.Uni` method, we should use them

Reply:
make sense